sdk/monitor/azure-monitor-opentelemetry/samples/tracing/http_openai.py

```diff
@@ -0,0 +1,30 @@
+from opentelemetry.instrumentation.httpx import HTTPXClientInstrumentor
+
+# The HTTPX instrumentation captures http requests made in the OpenAI library.
+# For openai>=1.4.0, it is important to instrument httpx before importing openai.
+HTTPXClientInstrumentor().instrument()
+
+# import openai # For openai==0.x
+from openai import OpenAI # For openai==1.x
+from opentelemetry.instrumentation.openai import OpenAIInstrumentor
+
+from azure.monitor.opentelemetry import configure_azure_monitor
+
+configure_azure_monitor()
+
+# The OpenAI Instrumation is not officially part of the OpenTelemetry community.
+# Therefore, we cannot atest to its stability. Use with appropiate caution.
+OpenAIInstrumentor().instrument()
```

Comment:
The httpx instrumentation also captures telemetry from openai. So, if we want to avoid using this library, I'd be open to just using httpx.